changelogs/fragments/12-correct_pull_wo_starting.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- "docker-compose - fixed not pull if ``state: present`` and ``stopped: true`` (https://github.com/ansible-collections/community.docker/issues/12, https://github.com/ansible-collections/community.docker/pull/119)."
```

I don't feel like my suggestion is well yet, please improve it any way you fit.
```suggestion
- "docker-compose - fix not pulling when ``state: present`` and ``stopped: true`` (https://github.com/ansible-collections/community.docker/issues/12, https://github.com/ansible-collections/community.docker/pull/119)."
```